django/pyproject.toml

```diff
@@ -25,23 +25,23 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
-python = "~3.11.0"
-django = ">=4.1,<4.2"
-psycopg2 = "~2.9.3"
-redis = { version = "~4.3.4", extras = [ "hiredis" ] }
+python = ">=3.11.8,<4"
+django = ">=5.0.3,<5.1"
+psycopg2 = "~2.9.9"
+redis = { version = "~5.0.3", extras = [ "hiredis" ] }
 uritemplate = "^4.1.1"
 iso3166 = "^2.1.1"
-rio-tiler = "5.0.3"
+rio-tiler = "5.0.3" # TODO: upgrade blocked on pydantic 2
 mercantile = "^1.2.1"
-django-ninja = "^0.21.0"
-celery = "^5.2.7"
-django-extensions = "^3.2.1"
+django-ninja = "~0.22.2" # TODO: upgrade blocked on pydantic 2
+celery = "^5.3.6"
+django-extensions = "^3.2.3"
 pillow = "^9.5.0"
```

Comment:
Note - we also need to upgrade Pillow, but that one actually seems to have major API changes. So, I'll make a follow up PR that does that one by itself.